src/components/SelectionList/ChatListItem.tsx

```diff
@@ -67,9 +67,11 @@ function ChatListItem<TItem extends ListItem>({
     const pressableStyle = [
         styles.selectionListPressableItemWrapper,
         styles.textAlignLeft,
+        styles.ph9,
```

Comment:
I think we only need to remove the margin, no need to change the padding
```suggestion
```

Reply:
@luacmartins that would decrease the spacing. We had ph4 + mh5 so if we set margin to 0 and want to maintain the previous spacing we need to include the previous margin to the current padding. In that way we will fix this issue maintaining the previous spacing.

Reply:
Yea, I think the spacing in this branch is wrong. Take a look at the images below:

Production
![prod](https://github.com/user-attachments/assets/e2957e92-d217-4e03-9e68-9e07242057ac)

This branch 
![Screenshot 2024-12-13 at 3 50 33 PM](https://github.com/user-attachments/assets/d6cd2d7e-2878-417c-a30b-55099dd07536)

There's extra space there

Reply:
My bad! Your correct this is because backgrounds are not applied to margins 👍